react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+function buttonColour(numberClicks) {
+  if (numberClicks >= 3 && numberClicks < 6) {
+    return 'purple';
+  } else if (numberClicks >= 6 && numberClicks < 9) {
+    return 'light-purple';
+  } else if (numberClicks >= 9 && numberClicks < 12) {
+    return 'coral';
+  } else if (numberClicks >= 12 && numberClicks < 15) {
+    return 'orange';
+  } else if (numberClicks >= 15 && numberClicks < 18) {
+    return 'yellow';
+  } else if (numberClicks >= 18) {
+    return 'white-background';
+  }
+}
```

Comment:
great idea using a function to get the class name. I would recommend making this a method in the HotButton class, you want all the functionality self contained and built into the component.